src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -432,18 +432,26 @@ public void Invoke(ScriptInvocationContext context)
 
         internal void InvokeResponse(InvocationResponse invokeResponse)
         {
-            if (_executingInvocations.TryRemove(invokeResponse.InvocationId, out ScriptInvocationContext context)
-                && invokeResponse.Result.IsSuccess(context.ResultSource))
+            if (_executingInvocations.TryRemove(invokeResponse.InvocationId, out ScriptInvocationContext context))
             {
-                IDictionary<string, object> bindingsDictionary = invokeResponse.OutputData
+                if (invokeResponse.Result.IsSuccess(context.ResultSource))
+                {
+                    IDictionary<string, object> bindingsDictionary = invokeResponse.OutputData
                     .ToDictionary(binding => binding.Name, binding => binding.Data.ToObject());
 
-                var result = new ScriptInvocationResult()
+                    var result = new ScriptInvocationResult()
+                    {
+                        Outputs = bindingsDictionary,
+                        Return = invokeResponse?.ReturnValue?.ToObject()
+                    };
+                    context.ResultSource.SetResult(result);
+                }
+                else
                 {
-                    Outputs = bindingsDictionary,
-                    Return = invokeResponse?.ReturnValue?.ToObject()
-                };
-                context.ResultSource.SetResult(result);
+                    Exception invocationException = invokeResponse.Result.InvocationResultException();
+                    context.Logger.LogError(invocationException.Message);
```

Comment:
Should you call LogError(invocationException, ...) here so the logger gets the full exception + stack?